tests/phpunit/tests/mail/test-phpmailer-translations.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class Test_PHPMailer_Translations extends WP_UnitTestCase {
```

Comment:
Could we add a test that actually loads a translation file for those new strings and then triggers a PHPMailer error, so that we can verify the error messages are actually translated?

Reply:
Let me know if you'd need help with those btw